pkg/scheduler/scheduler.go

```diff
@@ -114,3 +123,83 @@ type VPod interface {
 
 	GetResourceVersion() string
 }
+
+type ScaleCache struct {
+	lock                 sync.RWMutex //protects access to entries, entries itself is concurrency safe, so we only need to ensure that we correctly access the pointer
```

Comment:
Let's be specific on what this lock is associated with, usually `<name> + Mu`
```suggestion
	entriesMu                 sync.RWMutex // protects access to entries, entries itself is concurrency safe, so we only need to ensure that we correctly access the pointer
```